### ISSUE TITLE
Stop moving the bins in the accessible loo

### ISSUE THREAD

**Issue by Robotorium** [DoESLiverpool/somebody-should#1076]
I, too, enjoy letting bins roam free where they will. Nature is beautiful and, where possible, should be unrestricted. However, it would be nice if the bins in the accessible loo could stop being free-range.

Somehow they keep on wandering into the space between the changing table and the door, rendering the loo inaccessible because that's where wheelchairs go to get inside. That very much includes the little corner between the hand-dryer and the door, which would be a prime bit of bin nesting area if not for the fact it means I literally can't get in.

I'm not fussed about where they get corralled. I don't wish to cause them unhappiness. Personally, I quite like the nappy bin next to the painted brick wall where it's in easy reach for nappy changes, and the grey one under the sink, but I am entirely open to suggestion.

Just stop them from grazing where I'm going to have to be to actually use the loo, please.

Adrian has suggested we indicate preferred bin location on the floor somehow, and I'm perfectly happy to help with that. I just want to not crush them/bash my feet into them/get stuck trying to grab them whilst also rolling in (the latter is a fun trick in a manual chair that requires both hands to drive, but if I'm going to do tricks I require an audience, which I would prefer not to have when using the loo).

**Comment by johnmckerrell** [comment]
I wouldn't want to put anything on the floor as it's likely to just get dirty and look bad. I suspect it's the cleaner that's moved them so possibly just educating her would be enough. We could also add signs on the wall to indicate that the entrance path, or the floor in general, should be kept clear but I'm wondering if educating the cleaner and keeping an eye on it would be enough?

**Comment by DoESsean** [comment]
Has this been an issue since being raised, @Robotorium? If so, we'll need to look into a better way to ensure that it gets solved.

**Comment by amcewen** [comment]
I assume this isn't a problem now?  Please reopen it @Robotorium if that's not the case!
